lang/en.json

```diff
@@ -0,0 +1,53 @@
+{
+  "search-placeholder": "Search websites by name, URL or method (e.g. 2fa:sms)",
+  "categories": "Categories",
+  "backup": "Backup and Sync",
+  "banking": "Banking",
+  "betting": "Betting",
+  "cloud": "Cloud Computing",
+  "communication": "Communication",
+  "creativity": "Creativity",
+  "crowdfunding": "Crowdfunding",
+  "cryptocurrencies": "Cryptocurrencies",
+  "developer": "Developer",
+  "domains": "Domains",
+  "education": "Education",
+  "email": "Email",
+  "entertainment": "Entertainment",
+  "finance": "Finance",
+  "food": "Food",
+  "gaming": "Gaming",
+  "government": "Government",
+  "health": "Health",
+  "hosting": "Hosting/VPS",
+  "hotels": "Hotels and Accommodations",
+  "identity": "Identity Management",
+  "insurance": "Insurance",
+  "investing": "Investing",
+  "iot": "IoT",
+  "legal": "Legal & Compliance",
+  "marketing": "Marketing & Analytics",
+  "payments": "Payments",
+  "postal": "Post and Shipping",
+  "remote": "Remote Access",
+  "retail": "Retail",
+  "security": "Security",
+  "social": "Social",
+  "task": "Task Management",
+  "tickets": "Tickets and Events",
+  "transport": "Transport",
+  "universities": "Universities",
+  "utilities": "Utilities",
+  "vpn": "VPN",
+  "other": "Other",
```

Comment:
Should we set the translated category names in the API instead?

Reply:
Sure, but that would mean there would have to be a second translation source location.
And the translation would need to be fetched externally alongside the <category>.json.

Reply:
Why not embed them in the frontend API files? We can generate them as something like `categories.json` (default), `categories.fr.json`, `categories.sv.json`, etc., then update the `Categories` component's constructor to pull the appropriate one.

Reply:
That's certainly one way of doing it.
I rather prefer keeping all the translations in one single place, but if you'd like, I can implement your idea :shrug:

Reply:
Alright. Let's keep them here then.